src/code42cli/cmds/devices.py

```diff
@@ -0,0 +1,449 @@
+    backup_set_formatter = OutputFormatter(format, _backup_set_keys_map())
+    device_info = _get_device_info(state.sdk, device_guid)
+    formatter.echo_formatted_list([device_info])
+    click.echo()
+    backup_set_formatter.echo_formatted_list(device_info["backupUsage"])
+
+
+def _device_info_keys_map():
+    return {
+        "name": "Name",
+        "osHostname": "Hostname",
+        "guid": "GUID",
+        "status": "Status",
+        "lastConnected": "Last Connected Date",
+        "productVersion": "Code42 Version",
+        "osName": "Operating System",
+        "osVersion": "Operating System Version",
+    }
+
+
+def _backup_set_keys_map():
+    return {
+        "targetComputerName": "Destination",
+        "lastBackup": "Last Backup Activity",
+        "lastCompleted": "Last Completed Backup",
+        "archiveBytes": "Archive Size in Bytes",
+        "archiveGuid": "Archive GUID",
+    }
+
+
+def _get_device_info(sdk, device_guid):
+    return sdk.devices.get_by_guid(device_guid, include_backup_usage=True).data
+
+
+def _get_key_from_list_of_dicts(key, list_of_dicts):
+    return [item[key] for item in list_of_dicts]
+
+
+active_option = click.option(
+    "--active",
+    required=False,
+    type=bool,
+    is_flag=True,
+    default=None,
+    help="Get only active or deactivated devices. Defaults to getting all devices.",
+)
+
+org_uid_option = click.option(
+    "--org-uid",
+    required=False,
+    type=str,
+    default=None,
+    help="""Limit devices to only the ones in the org you specify.
+    Note that child orgs will be included.""",
+)
+
+include_usernames_option = click.option(
+    "--include-usernames",
+    required=False,
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="Add the username associated with a device to the output.",
+)
+
+
+@devices.command(name="list", help="Get information about many devices")
+@active_option
+@click.option(
+    "--days-since-last-connected",
+    required=False,
+    type=int,
+    help="Return only devices that have not connected in the number of days specified.",
+)
+@org_uid_option
+@click.option(
+    "--drop-most-recent",
+    required=False,
+    type=int,
+    help="""Will drop the X most recently connected devices for each user from the
+    result list where X is the number you provide as this argument. Can be used to
+    avoid passing the most recently connected device for a user to the deactivate command.""",
+)
+@click.option(
+    "--include-backup-usage",
+    required=False,
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="""Return backup usage information for each device
+    (may significantly lengthen the size of the return).""",
+)
+@include_usernames_option
+@click.option(
+    "--include-settings",
+    required=False,
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="""Include device settings in output.""",
+)
+@format_option
+@sdk_options()
+def list(
```

Comment:
This should be renamed to `_list` so we don't shadow the built-in `list()`.

Reply:
Renamed it to `list_devices`, to match the other `list_backup_sets` method.